src/proxy/nginx.conf.template

```diff
@@ -30,10 +32,17 @@ http {
       server ${FRONTEND_HOST}:${FRONTEND_PORT};
   }
 
+XXX
```

Comment:
This works but it's really ugly. Let's use standard [nginx include directive](http://nginx.org/en/docs/ngx_core_module.html#include) to include all files in a directory (e.g. `include /etc/nginx/includes/*.conf`) and copy dev configuration to the includes folder if and only if `$IS_DEV_ENV == 1`.
Both conditional blocks can be in the same file if I didn't overlook something.

Reply:
Unfortunately, `upstream`s must be in `http` section and `location`s go under `server`, so  I had to split it into two files. But I believe the new way I did it is much more flexible.